kcidb/test_oo.py

```diff
@@ -217,6 +217,25 @@ def query_str(self, pattern_string):
             kcidb.orm.Pattern.parse(pattern_string)
         )
 
+    @staticmethod
+    def filter_valid(container, type):
+        """
+        Filter out the valid items in a container.
+        Args:
+            container:  A dictionary whose values are object(s) that
+                        have "valid" property.
+            type:       A key in the container.
+
+        Returns:
+            The list of object(s) with valid property which is true.
```

Comment:
This would be easier to read, I think:
```suggestion
            The list of object(s) with valid property true.
```